<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_three_point</class>
 <widget class="QDialog" name="Dialog_three_point">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>331</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Fill in the mapping locations of the following
three points.
Notes:
1.Input range is limited to [0,1].
2.The following default values correspond
to the coordinates in the original image, and
the values are modified to map to the desired
coordinates.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>    (0,0) =&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string> (0,width) =&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>(height,0) =&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>*width,</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>*height</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>*height</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>*height</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>*width,</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>*width,</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_three_point</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_three_point</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
